### PR TITLE
bridge: cleanup of c_types functions

### PR DESCRIPTION
**Pull request by goaway**
Signed-off-by: Mike Schore <mike.schore@gmail.com>

### CODE REVIEW

File: library/common/types/c_types.h [hunk 1]
```diff
@@ -159,36 +159,36 @@ void* safe_malloc(size_t size);
...
  * Called by a receiver of envoy_data to indicate memory/resources can be released.
  * @param data, envoy_data to release.
  */
 void release_envoy_data(envoy_data data);
 
 /**
- * Called by a receiver of envoy_headers to indicate memory/resources can be released.
- * @param headers, envoy_headers to release.
+ * Called by a receiver of envoy_map to indicate memory/resources can be released.
+ * @param data, envoy_map to release.
```
Comment (Augustyniak) [inline library/common/types/c_types.h:170]:
```suggestion
 * @param map, envoy_map to release.
```

**Review by Augustyniak — APPROVED** [review]
Nice! one nit comment

File: library/common/types/c_types.h (hunk 1 above)
Comment (goaway) [reply library/common/types/c_types.h:170]: Thanks, good catch :)

**Comment by junr03** [comment]
looks like the unit tests timed out. Kicked it

**Review by junr03 — APPROVED** [review]

**Comment by goaway** [comment]
Thanks @Augustyniak and @junr03!

**Pull request merged**